### DISCUSSION TITLE
Uğraştığımız boşa gidecek ...

### DISCUSSION THREAD

**Discussion by coinsinov** [IBAX-io/go-ibax#1472] [Soru Cevap - Türkçe]
Adminler, hele de İBAX Türkiye Telegram kanalı adminleri insanlara yardımcı olmuyor, olamıyor bir şey bildikleri yok. Sorularınıza cevap veremiyor yardımcı olamaya çalışanları da kanaldan banlıyorlar. Bence günlük baktığımız kastığımız işler boşa gidecek. İngilizce kanalı durmadan etkinlik yapıyor. Yarın elimizde hiç bir şey olmayacak.

**Comment by Skips1** [comment]
Bunda haklısın bu konuyu ben daha önce kaleme aldım ama gene kimse beni dikate almadı umarım buna karşı çıkacak bir yetkili olur :(

**Comment by Yusufknk** [comment]
İbax'ın birden çok sosyal medya kanalları var. Bunun için çalışıyoruz. Söylediklerinizi dikkate alacağız.
